upgrading/change-log.md

```diff
@@ -7,6 +7,39 @@ heading: Version 3.15.x
 
 <p class="release-date">2020</p>
 
+<h4>Fixes</h4>
+
+- In Marimekko chart, using `yAxisValueDecimal` now works properly when `setAdaptiveYMin` and `forceYAxisValueDecimals` are enabled.
+
+- Decimal and thousand separator attributes now work properly on Marimekko chart. 
+
+- For all 2D and 3D bar chart variants, the fontSize attribute now works properly when set with a numeric value. Earlier a JS error was thrown.
```

Comment:
I think it should be `For all 2D and 3D column and bar chart variants, the `fontSize` and `labelFontSize` attributes now work properly when set to a string and numeric value. Earlier a JS error was thrown when numeric value was set.

Reply:
changed